packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -12,14 +12,39 @@ export interface InitializeNodeEnvironmentOptions extends InitializerOptions {
     processOptions?: Pick<SpawnOptions, 'cwd' | 'shell' | 'env' | 'stdio' | 'windowsHide'>;
 }
 
+/**
+ * Detailed info about process exit.
+ */
+export interface ProcessExitDetails {
+    /**
+     * The exit code if the child exited on its own
```

Comment:
The comment makes me question what "on its own" means. Not having a comment would make this absolutely clear.

Reply:
this is taken from node documentation about child exit event